plugins/modules/zos_copy.py

```diff
@@ -3140,7 +3140,8 @@ def normalize_line_endings(src, encoding=None):
         src_tag = encoding["from"]
 
     if src_tag != "IBM-037":
-        fd, converted_src = tempfile.mkstemp()
+        # fd, converted_src = tempfile.mkstemp(dir=os.environ['TMP'])
```

Comment:
We could get rid of this identical commented line

Reply:
done